### ISSUE TITLE
R Markdown Résumé/CV templates :construction_worker:

### ISSUE THREAD

**Issue by mitchelloharawild** [ropensci/ozunconf18#24]
## Résumés for R Markdown
I've had my résumé as a R Markdown template for a few months now, and @robjhyndman has recently converted his CV too (https://github.com/robjhyndman/CV).

I think using R Markdown for CVs is a great match. This is especially true for listed elements (education, experience, accolades) which can be filled in from an external source (or perhaps filtered by tags).

## Possible features

My hopes and dreams for a **R**ésumés package would include these features:
- A variety of templates ([list of popular open source resumes templates](https://github.com/search?o=desc&q=latex+resume&s=stars&type=Repositories))
- Functions to produce listed sections (education, experience, accolades)
- Functions/add-in(s) to maintain a résumé contents dataset
- Tagging system for entries, to support generation of tag-specific résumés

## Potential name(s)
``` r
available::available("CV")
#> ── CV ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
#> Name valid: ✔
#> Available on CRAN: ✔ 
#> Available on Bioconductor: ✔
#> Available on GitHub:  ✖ 
#> Bad Words: ✔
#> Abbreviations: http://www.abbreviations.com/CV
#> Wikipedia: https://en.wikipedia.org/wiki/CV
#> Wiktionary: https://en.wiktionary.org/wiki/CV
#> Urban Dictionary:
#>   1. a [curriculum vitae] or resume is a list of your  [qualifications], achievements, skills etc that you give to a  [prospective] employer.  2. tits.
#>   http://cv.urbanup.com/3007232
#> Sentiment:???
```

<sup>Created on 2018-11-13 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by goldingn** [comment]
Awesome idea! The one thing I'd really like is a way to pull papers from Google Scholar, format them nicely in a publications section, and possibly add citation counts as a column on the right.

**Comment by goldingn** [comment]
(though based just on the name, I would assume a package called CV was about cross validation)

**Comment by goldingn** [comment]
And also CRAN/GitHub packages with rstudio CRAN mirror download metrics

**Comment by mitchelloharawild** [comment]
All good suggestions.
For pulling papers from Google Scholar and packages from CRAN, it'd be nice and extensible if the package made it easy to include data from other sources. That way scholar entries can be obtained using the [scholar package](https://github.com/jkeirstead/scholar), cran downloads can come from the [cranlogs package](https://github.com/metacran/cranlogs)

Not sure about the name, short and simple is always nice, but it is important to avoid confusion. "resume" also has similar meaning conflicts. Open to suggestions for names.

**Comment by goldingn** [comment]
Sweet!

`vitae` for package name? A bit obscure maybe

**Comment by will-r-chase** [comment]
This sounds like a really cool idea. I recently made a CV template with HTML/CSS (https://github.com/will-r-chase/CV_template). Since Rmd just knits to HTML/CSS it might be possible to use some of these ideas to style and format the output. Might also be possible to use a Xaringan-esque interface so users could set different CSS selectors for different elements of the Rmd document? 

Alternatively I'm sure there's some LaTeX templates, but I'm not too familiar with that. Anyways, I'd be happy to contribute with some custom CSS/HTML/styling integration if needed. The options to automatically style things and pull references from the web would really be big advantages to just doing your CV manually.

**Comment by mitchelloharawild** [comment]
I had LaTeX templates in mind when thinking about this project, however I know that there are also plenty of great web-based CVs available. It'd be nice to keep a relatively consistent interface across the templates (to allow minimal friction when trying different templates), but I'm sure we can integrate web CVs into the package too.

**Comment by Lingtax** [comment]
@ekothe has existing code to help with the data mining and integration.

**Comment by raymondben** [comment]
This is a pretty cool idea. I did my CV using org-mode in emacs for years, and recently migrated across to Rmd with a `tufte` style template. I like the fact that this package could provide both content helpers (to pull stuff from Scholar/GitHub/researchgate/CRAN/whatever), as well as style (templates).

**Comment by smwindecker** [comment]
Great idea! Especially like the option to add a 'software' section that can pull info from Github.

**Comment by ekothe** [comment]
It's also possible to take some information from ORCID - including education, affiliation, etc. if people have a complete profile there.

**Comment by mitchelloharawild** [comment]
Definitely! For this, users could use the [rorchid](https://github.com/ropensci/rorcid) package in their R chunks to produce (or add to) resume sections.

**Comment by emitanaka** [comment]
It might be worthwhile thinking of looking at `pagedown`: https://github.com/rstudio/pagedown in particular with `pagedown::html_resume`. I do like the idea of `software` section from github info that @smwindecker mentions with a possible auto update of stars or forking.

**Comment by emitanaka** [comment]
Another thing is that it will be nice to have a website much like overleaf [here](https://www.overleaf.com/latex/templates) where it is easy for people to submit their own templates with tags.

**Comment by ekothe** [comment]
In an Australian context it might be interesting/useful to see if we can develop templates that are consistent with (current) ARC/NHMRC templates (insofar as those schemes still require Chief Investigator resumes).

**Comment by agbarnett** [comment]
Thanks Nick. In terms of the ARC/NHMRC, I have heard that they may both soon finally switch to using an automatically generated publication lists using ORCID.
